src/main/resources/application.properties

```diff
@@ -1 +1,2 @@
-spring.application.name=wemove
\ No newline at end of file
+spring.application.name=wemove
+spring.profiles.active=prod
```

Comment:
please delete this line.
you can dig further about spring profiles in spring boot documentation 
https://docs.spring.io/spring-boot/reference/index.html